### PR TITLE
STEAM-3857 Display config location after configuration setup

### PR DESCRIPTION
**Pull request by Rody-Kirwan**
Adds config location info to configure command output

## Proposed Changes

  - ON successful configuration inform user of config location
  -
  -

### CODE REVIEW

File: src/commands/configure.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+class ConfigureCommand extends BaseCommand {
   async run() {
     const prompts = getPrompts(['swaggerHubUrl','apiKey'])(getConfig())
+    const { configDir } = this.config
 
-    inquirer.prompt(prompts).then(setConfig)
+    inquirer.prompt(prompts)
+      .then(setConfig)
+      .then(this.logCommandSuccess({ configDir }))
+      .catch(console.log)
```
Comment (Rody-Kirwan) [inline src/commands/configure.js:14]: Updating this...

File: src/template-strings/errors.js [hunk 2]
```diff
@@ -3,11 +3,13 @@ const { wrapTemplates } = require('../utils/general')
...
   ApiCreate: 'API version \'{{owner}}/{{name}}/{{version}}\' already exists in SwaggerHub',
 
+  Configure: 'Failed to write config to {{configDir}}',
+
   argsMustMatchFormat: 'Argument must match {{format}} format',
 
   cannotParseDefinition: 'There was a problem with parsing {{fileName}}. Ensure the definition is valid. {{e}}',
 
-  cannotParseOasVersion: 'Cannot determine OAS version from file',
+  cannotParseOasVersion: 'Cannot determine OAS versio n from file',
```
Comment (barryadk) [inline src/template-strings/errors.js:12]: typo here

File: test/commands/configure.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const { expect, test } = require('@oclif/test')
+const ConfigureCommand = require('../../src/commands/configure')
+const inquirer = require('inquirer')
+const { setConfig } = require('../../src/config')
+
+describe('successful configuration', () => {
+  test
+    .stub(ConfigureCommand, 'config', {
+      configDir: 'pio'
```
[the remark this answers is not in the document]
Comment (Rody-Kirwan) [reply test/commands/configure.test.js:9]: No - unnecessary leftovers from battling with a failing unit test.
Will update soon

**Comment by barryadk** [comment]
Should the message display the output config file? Currently it displays the config directory:
`Config saved to C:\Users\barry\AppData\Local\swaggerhub-cli`
instead of
`Config saved to C:\Users\barry\AppData\Local\swaggerhub-cli\config.json`

**Comment by Rody-Kirwan** [comment]
> Should the message display the output config file? Currently it displays the config directory:
> `Config saved to C:\Users\barry\AppData\Local\swaggerhub-cli`
> instead of
> `Config saved to C:\Users\barry\AppData\Local\swaggerhub-cli\config.json`

Yes - good catch @barryadk - will update

File: src/commands/configure.js [hunk 4]
```diff
@@ -6,12 +6,12 @@ const { setConfig, getConfig } = require('../config')
 class ConfigureCommand extends BaseCommand {
   async run() {
     const prompts = getPrompts(['swaggerHubUrl','apiKey'])(getConfig())
-    const { configDir } = this.config
+    const configFilePath = `${this.config.configDir}/config.json`
```
Comment (barryadk) [inline src/commands/configure.js:9]: Wrong path separator for Windows clients.
`Config saved to C:\Users\barry\AppData\Local\swaggerhub-cli/config.json`

Comment (Rody-Kirwan) [reply src/commands/configure.js]: Updated :D

File: src/commands/configure.js [hunk 5]
```diff
@@ -6,12 +6,12 @@ const { setConfig, getConfig } = require('../config')
 class ConfigureCommand extends BaseCommand {
   async run() {
     const prompts = getPrompts(['swaggerHubUrl','apiKey'])(getConfig())
-    const configFilePath = `${this.config.configDir}/config.json`
```
Comment (barryadk) [inline src/commands/configure.js]: Could use https://nodejs.org/api/path.html#path_path_sep to print the full path?

**Pull request merged**